docs/general/administration/hardware-selection.md

```diff
@@ -43,16 +43,12 @@ Intel ARC GPUs are recommended when upgrading an existing system to be used as a
 
 :::
 
-:::tip Fully Utilizing Intel-based Macs
-
-It is recommended that Intel-based macs be used with Windows or Linux installed to host Jellyfin. Many hardware acceleration features aren't available on MacOS due to the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork not being available.
-
-:::
-
 ### Low Power Applications
 
 For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended. It is also recommended that [Low Power Encoding](/docs/general/administration/hardware-acceleration/intel/#low-power-encoding) be setup.
 
+Alternatively, you can use an Apple Silicon Mac for even lower power consumption.
```

Comment:
Doubt. Does an Apple Silicon Mac Mini consume measurably less power than an N100 based mini PC?

Reply:
N100-based mini PCs are less power-efficient than you might expect. In fact, many of them have higher power consumption idling than an M1 Mac mini during transcoding due to poor firmware.